dev/custom-pages/internet-privacy.html

```diff
@@ -0,0 +1,39 @@
+<div class="container">
```

Comment:
`.container` removed. Note this was also what was causing the misalignment Keegan mentioned in teamwork.